server/plugin.go

```diff
@@ -888,6 +893,15 @@ func (p *Plugin) IsRemoteUser(user *model.User) bool {
 	return user.RemoteId != nil && *user.RemoteId == p.remoteID
 }
 
+func (p *Plugin) IsUserConnected(userID string) (bool, error) {
+	token, err := p.store.GetTokenForMattermostUser(userID)
+	if err != nil && err != sql.ErrNoRows {
```

Comment:
In the store, if the user has no encrypted token, we return an error:

https://github.com/mattermost/mattermost-plugin-msteams/blob/ae3640f39f667d0d03134cf46065da7124a307f8/server/store/sqlstore/store.go#L364-L366

This is a bit weird, I think we should either change the store to return `nil, nil` (no token but no error), or we should make it a proper `ErrNoTokenFound`, and this method could check if `errors.Is(err, ErrNoTokenFound)` to return false rather than an error.

Reply:
I updated to return 'nil, nil', as most places that call this function don't even check the error that is returned, and it didn't cause a bunch of unit test failures. If we want to return a proper, `ErrNoTokenFound`, we should probably handle that in a separate PR.